services/protocols/MS-Graph-API.js

```diff
@@ -51,8 +52,10 @@ async function _sendMail(email, count) {
   await sendOrRetry(async () => {
     const client = Client.initWithMiddleware({
       authProvider: MS_GRAPH_API_AUTH_PROVIDER,
+      debugLogging: NODE_ENV === "development",
```

Comment:
Better to work with `LOG_*` flags as done on other services (eg. [mu-authorization](https://github.com/mu-semtech/mu-authorization#logging), [acmidm-login-service](https://github.com/kanselarij-vlaanderen/acmidm-login-service#configuration)). We may want to enable the flag temporary on a production environment without the hassle to setup the service in develoment mode.